packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -416,7 +404,10 @@ export default {
 
         dialogMessage () {
             return {
-                name: 'error-dialog'
+                name: 'error-dialog',
+                props: {
+                    'redirect-url': this.redirectUrl
```

Comment:
Wouldn't `redirectUrl` work instead of `redirect-url`? I'm not sure, but it's worth asking.

Reply:
@SachaZvetelman I tested it now and it works with `redirectUrl` as well.